addon/templates/components/md-nav.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="nav-wrapper">
+  <div class="container">
```

Comment:
This container is bringing some additional styling with it that I'm not sure is wanted.  I don't think this container is needed—it is not in any of the materialize navbar examples.